### DISCUSSION TITLE
Serwer SQL VM Problem

### DISCUSSION THREAD

**Discussion by xluciola** [AzureCommunityPL/group#138] [Pomoc]
Miałem problem z połączeniem mojego programu do SQL database i za radą kolegów z grupy zainstalowałem SQL server na VM - ponieważ nie ma zasobów dla mnie dla SQL managed instances ( nie wiem czemu) .

Virtual Machine-  Stoi a SQL virtual machine w panelu mam Offline z komunikatem :"The SQL virtual machine resource is not in a valid state for management. The cause could be the SQL virtual machine is in bad states, the underlying virtual machine state is invalid (Example: stopped, deallocated or failed) or the virtual machine was not found. Please make sure the SQL virtual machine and the underlying virtual machine are healthy."

połączyłem się pulpitem zdalnym z Maszyną Virtualną i serwer jest zainstalowany.
jak mam połączyć się z serwerem i bazą danych z moim programem który chce z bazy skorzystać ??

dziękuję

**Comment by eklime** [comment]
A port 1433 odblokowałeś na świat? (co nie jest w żadnym stopniu zalecanym rozwiązaniem)
